consume_messages.rb

```diff
@@ -0,0 +1,38 @@
+require 'aws-sdk'
+
+# Bring environment variables in ./config/.env into scope.
```

Comment:
We can use the [dotenv gem](https://github.com/bkeepers/dotenv) gem in development as an easy way to
set the environment variables that are used in `settings.yml`.